packages/test-app/src/hooks/useFetchBySlug.ts

```diff
@@ -0,0 +1,33 @@
+import { ExternalSDKMode, useFetchExperience } from '@contentful/experience-builder';
```

Comment:
This hook is temporary and will be taken out once the PR to merge in the `fetchBySlug` hook is in for the sdk